x-pack/test/api_integration/apis/management/index_management/data_streams.ts

```diff
@@ -210,6 +210,25 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
+    describe('Update', () => {
```

Comment:
Can we add tests for serverless as well 🙏

Reply:
I've added functional tests for serverless, but we might get hit by the same problem the other index_management functional tests have now that make them flaky! Hopefully those will be sorted out soon 😀